tests/jballerina-integration-test/src/test/resources/grpc/src/tool/helloWorldGateway.proto

```diff
@@ -0,0 +1,46 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Reply:
I made the change to the file and the other test cases I added as well